zio-http/src/main/scala/zhttp/service/Client.scala

```diff
@@ -175,4 +200,10 @@ object Client {
   object Config {
     def empty: Config = Config()
   }
+
+  val zioHttpVersion: String           = Client.getClass().getPackage().getImplementationVersion()
```

Comment:
- Use `CharSequence` instead of string
- Use `AsciiString.cached` to generate the char seqeuence.